docs/src/pages/components/SkipLink/WebSkipLinkPage.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import BpkSkipLink from 'bpk-component-skip-link';
+import skipLinkReadme from 'bpk-component-skip-link/README.md';
+import { cssModules } from 'bpk-react-utils';
+
+import { WebComponentPage } from '../../../components/ComponentPage';
+import Paragraph from '../../../components/Paragraph';
+
+import STYLES from './skip-link-page.scss';
+
+const getClassName = cssModules(STYLES);
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: [
+      <Paragraph>
+        Skip links benefit sighted keyboard or switch users by providing them
+        with a way to skip over lengthy menus or filters. Skip links should be
+        absolute-positioned to ensure that when they become visible they do not
```

Comment:
```suggestion
        absolutely positioned to ensure that when they become visible they do not
```

No hyphen required and should be 'absolutely' as it's being used as an adjective.